### PR TITLE
resolvectl: add page

### PR DESCRIPTION
**Pull request by 258204**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/resolvectl.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# resolvectl
+
+> Resolve domain names, IPV4 and IPv6 addresses, DNS resource records, and services.
+> Introspect and reconfigure the DNS resolver.
```
Comment (CleanMachine1) [inline pages/linux/resolvectl.md:4]: I don't think introspect is the correct word here.

File: pages/linux/resolvectl.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Show DNS settings:
+
+`resolvectl status`
+
+- Resolve the IPv4 and IPv6 addresses for one or more domains:
+
+`resolvectl query {{domain}}`
+
+- Retrieve the domain of {{IP}}:
```
Comment (CleanMachine1) [inline pages/linux/resolvectl.md:15]:
```suggestion
- Retrieve the domain of a specified IP:
```

File: pages/linux/resolvectl.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`resolvectl query {{ip}}`
```
Comment (CleanMachine1) [inline pages/linux/resolvectl.md:17]:
```suggestion
`resolvectl query {{IP}}`
```

File: pages/linux/resolvectl.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- Retrieve MX record of {{domain}}:
```
Comment (CleanMachine1) [inline pages/linux/resolvectl.md:19]:
```suggestion
- Retrieve an MX record of {{domain}}:
```

File: pages/linux/resolvectl.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`resolvectl  --legend=no -t MX query  {{domain}}`
+
+- Resolve an SRV record, for example _xmpp-server._tcp gmail.com:
+
+`resolvectl service _{{service}}._{{protocol}} {{name}}`
+
+- Retrieve the public key from an email address with an OPENPGP Record:
```
Comment (CleanMachine1) [inline pages/linux/resolvectl.md:27]:
```suggestion
- Retrieve the public key from an email address with a PGP Record:
```

**Comment by CleanMachine1** [comment]
Good to see you contributing again!

File: pages/linux/resolvectl.md (hunk 1 above)
Comment (258204) [reply pages/linux/resolvectl.md:4]: From  the man page:
"Resolve domain names, IPV4 and IPv6 addresses, DNS resource records, and services; introspect and reconfigure the DNS resolver."

File: pages/linux/resolvectl.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+`resolvectl status`
+
+- Resolve the IPv4 and IPv6 addresses for one or more domains:
+
+`resolvectl query {{domain}}`
+
+- Retrieve the domain of a specified IP:
+
+`resolvectl query {{IP}}`
```
Comment (navarroaxel) [inline pages/linux/resolvectl.md:17]: We use `snake_case` for placeholders:
```suggestion
`resolvectl query {{ip_address}}`
```

File: pages/linux/resolvectl.md (hunk 5 above)
Comment (258204) [reply pages/linux/resolvectl.md:27]: I made a typo here. The DNS record type is called OPENPGPKEY, probably the longest one available. What do you think about changing it to that?

File: pages/linux/resolvectl.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+- Retrieve an MX record of {{domain}}:
```
Comment (navarroaxel) [inline pages/linux/resolvectl.md:19]: The token syntax here is not well rendered
```suggestion
- Retrieve an MX record of domain:
```

File: pages/linux/resolvectl.md (hunk 1 above)
Comment (258204) [reply pages/linux/resolvectl.md:4]: I believe it does support  [network introspection commands](https://www.ibm.com/docs/en/api-connect/5.0.x?topic=interface-network-introspection-commands), though I admit I didn't include introspection in the examples.

File: pages/linux/resolvectl.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+> Introspect and reconfigure the DNS resolver.
+> More information: <https://www.freedesktop.org/software/systemd/man/resolvectl.html>.
+
+- Show DNS settings:
+
+`resolvectl status`
+
+- Resolve the IPv4 and IPv6 addresses for one or more domains:
+
+`resolvectl query {{domain}}`
```
Comment (sbrl) [inline pages/linux/resolvectl.md:13]: Are domains space separated? Perhaps we could indicate this somehow?

File: pages/linux/resolvectl.md (hunk 5 above)
Comment (sbrl) [reply pages/linux/resolvectl.md:27]: What about this?
```suggestion
- Retrieve the public key from an email address from an OPENPGPKEY DNS record:
```

File: pages/linux/resolvectl.md (hunk 8 above)
Comment (navarroaxel) [reply pages/linux/resolvectl.md:13]: This way
```suggestion
`resolvectl query {{domain1 domain2 ...}}`
```

Comment (258204) [reply pages/linux/resolvectl.md:13]: Thanks @navarroaxel!

File: pages/linux/resolvectl.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+`resolvectl query {{domain1 domain2 ...}}`
+
+- Retrieve the domain of a specified IP:
+
+`resolvectl query {{ip_address}}`
+
+- Retrieve an MX record of domain:
+
+`resolvectl  --legend=no -t MX query  {{domain}}`
```
Comment (navarroaxel) [inline pages/linux/resolvectl.md:21]: We prefer long arguments, and there is extra spaces.
```suggestion
`resolvectl --legend={{no}} --type={{MX}} query {{domain}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by sbrl — APPROVED** [review]
Looks ok to me after @navarroaxel's comments have been addressed

File: pages/linux/resolvectl.md (hunk 9 above)
Comment (258204) [reply pages/linux/resolvectl.md]: I selected commit suggestion, yet it still says 1 change requested. Is there anything else that needs to be changed?

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**